### PR TITLE
Deploy to production

### PR DESCRIPTION
**Pull request by kumy**
Production is 60 commits behind rec.
If we're ok with current rec, then merging this PR will auto-deploy prod at 91d1259

### CODE REVIEW

**Comment by kumy** [comment]
I think we could safely ignore codacy report.

@boly38 Your review **is needed** before we can merge it to `prod` branch. (Those are the security I've defined for that branch + travis)

**Comment by boly38** [comment]
yep. will take a look tomorrow if okay

**Comment by boly38** [comment]
Database has to be up-to-date with new waypointy translation tables (cf docker/mariadb/gk*.sql) : script must be executed by docker

I dont see wrong things, ok

I pushed a branch "broken/contactEmail" that fix contact email image / but it's not autodeployed (stay on my WIP branch 'feature/kontact': this is old boly38 branch location)

File: website/lib/LDHelper.class.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    public function __construct($geokretyName, $geokretyUrl, $geokretyLogoUrl) {
+        $this->geokretyName = $geokretyName;
+        $this->geokretyUrl = $geokretyUrl;
+        $this->geokretyLogoUrl = $geokretyLogoUrl;
+
+        $this->geokretyLogo = LDGeneratorFactory::createLdContext('ImageObject', [
+            'url' => $geokretyLogoUrl,
+        ]);
+        /*
```
Comment (boly38) [inline website/lib/LDHelper.class.php]: I will remove this bloc comments in a future PR

**Comment by kumy** [comment]
> would like to test it using boly38 autodeploy but it's not autodeployed

I have replaced the `git rebase` by a `git reset --hard origin/boly38`, it should be fine now. Also `webhook` scripts and conf.example are now public https://github.com/geokrety/webhook

**Comment by kumy** [comment]
> Database has to be up-to-date with new waypointy translation tables (cf docker/mariadb/gk*.sql) : script must be executed by docker

Tables are already present on production database and contains those data (argh can't attach them here, I'll send you them via mail)

**Comment by kumy** [comment]
I think we're ready, let's test our first auto deployment to the prod!

**Comment by boly38** [comment]
ok cool.  files seems ok too.
(I don't know why you send them to me, but i check them quickly. ok.. / my note about that files was maybe useless ;) )

**Comment by kumy** [comment]
> I don't know why you send them to me

Sorry I should have been more precise. "Because there are more records in the files I gave you" :)

**Comment by kumy** [comment]
Auto deployed worked well! And the enhanced log page seems to work fine.

**Comment by boly38** [comment]
yep good job!💪💪

**Comment by boly38** [comment]
ld json seems valid too


example: 
https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fgeokrety.org%2Fkonkret.php%3Fid%3D55249

(thinking that maybe reference GKxxxx wixh is so important must be added to title/keywords..etc)

**Comment by kumy** [comment]
So nice!

> sameAs mypage.php?userid=34386

Shouln't it be a full url?

**Comment by boly38** [comment]
I think both are ok but not 100% sure
another thing on https://search.google.com the property is http:// one 
image

**Pull request merged**